packages/riipen-ui/src/components/Tab.test.jsx

```diff
@@ -0,0 +1,363 @@
+
+    try {
+      mount(<Tab value />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Tab value />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<Tab value />);
+
+      expect(wrapper.find("Tab").props().active).toEqual(false);
+      expect(wrapper.find("Tab").props().breakpoint).toEqual("sm");
+      expect(wrapper.find("Tab").props().color).toEqual("secondary");
+      expect(wrapper.find("Tab").props().disabled).toEqual(false);
+      expect(wrapper.find("Tab").props().fullWidth).toEqual(false);
+      expect(wrapper.find("Tab").props().orientation).toEqual("horizontal");
+    });
+  });
+
+  describe("active and displayActive props", () => {
+    const props = {};
+    const color = "primary";
+    const orientation = "horizontal";
+
+    const tests = [
+      {
+        active: false,
+        displayActive: false,
+        argument: `${color}-active`,
+        expected: false
+      },
+      {
+        active: true,
+        displayActive: false,
+        argument: `${color}-active`,
+        expected: true
+      },
+      {
+        active: false,
+        displayActive: true,
+        argument: `${color}-active`,
+        expected: true
+      },
+      {
+        active: true,
+        displayActive: true,
+        argument: `${color}-active`,
+        expected: true
+      },
+      {
+        active: false,
+        displayActive: false,
+        argument: `${orientation}-active`,
+        expected: false
+      },
+      {
+        active: true,
+        displayActive: false,
+        argument: `${orientation}-active`,
+        expected: true
+      },
+      {
+        active: false,
+        displayActive: true,
+        argument: `${orientation}-active`,
+        expected: true
+      },
+      {
+        active: true,
+        displayActive: true,
+        argument: `${orientation}-active`,
+        expected: true
+      }
+    ];
+
+    tests.forEach(test => {
+      it(`sets ${test.expected} class name with custom active and displayActive prop values`, () => {
+        props.active = test.active;
+        props.displayActive = test.displayActive;
+        props.color = "primary";
+        props.orientation = "horizontal";
+
+        const wrapper = mount(<Tab {...props} value />);
+
+        expect(
+          wrapper
+            .find("Tab")
+            .childAt(0)
+            .hasClass(test.argument)
+        ).toEqual(test.expected);
+      });
+    });
+  });
+
+  describe("breakpoint prop", () => {
+    describe("custom breakpoint and orientation props", () => {
+      const props = {};
+
+      const tests = [
+        {
+          breakpoint: "md",
+          orientation: "horizontal",
+          argument: "breakpoint",
+          expected: true
```

Comment:
I think you could maybe make this a bit simpler in tandem with what is below on lines 146-147. What if you defined it like:

```js
{
  props: {
    breakpoint: 'md',
    orientation: 'horizontal',
  },
  expected: true,
}
```

then  your test below could be:

```js
tests.forEach(test => {
  it(`${test.expected ? 'does' : 'does not'} set breakpoint class name with breakpoint of ${test.props.breakpoint} and orientation of ${test.props.orientation}`, () => {
    const wrapper = mount(<Tab {...test.props} value />);

    expect(
      wrapper
        .find("Tab")
        .childAt(0)
        .hasClass("breakpoint")
    ).toEqual(test.expected);
  });
})
```

Reply:
Adjusted, I like the use of the ternary operator in the test name